eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -0,0 +1,886 @@
+    TxBody
+      ( TxBody,
+        inputs,
+        collateral,
+        collateralAddresses,
+        outputs,
+        txcerts,
+        txwdrls,
+        txfee,
+        txvldt,
+        txUpdates,
+        reqSignerHashes,
+        mint,
+        scriptIntegrityHash,
+        adHash,
+        txnetworkid
+      ),
+    inputs',
+    collateral',
+    collateralAddresses',
+    outputs',
+    certs',
+    wdrls',
+    txfee',
+    vldt',
+    update',
+    reqSignerHashes',
+    mint',
+    scriptIntegrityHash',
+    adHash',
+    txnetworkid',
+    AlonzoBody,
+    EraIndependentScriptIntegrity,
+    ScriptIntegrityHash,
+  )
+where
+
+import Cardano.Binary
+  ( DecoderError (..),
+    FromCBOR (..),
+    ToCBOR (..),
+    decodeBreakOr,
+    decodeListLenOrIndef,
+    encodeListLen,
+  )
+import Cardano.Crypto.Hash
+import Cardano.Ledger.Address (Addr (..))
+import Cardano.Ledger.Alonzo.Data (AuxiliaryDataHash (..), Data, DataHash, hashData)
+import Cardano.Ledger.BaseTypes
+  ( Network (..),
+    StrictMaybe (..),
+    isSNothing,
+  )
+import Cardano.Ledger.Coin (Coin (..))
+import Cardano.Ledger.Compactible
+import Cardano.Ledger.Core (PParamsDelta)
+import qualified Cardano.Ledger.Core as Core
+import Cardano.Ledger.Credential (Credential (..), PaymentCredential, StakeReference (..))
+import qualified Cardano.Ledger.Crypto as CC
+import Cardano.Ledger.Era (Crypto, Era)
+import Cardano.Ledger.Hashes
+  ( EraIndependentScriptIntegrity,
+    EraIndependentTxBody,
+  )
+import Cardano.Ledger.Keys (KeyHash (..), KeyRole (..))
+import Cardano.Ledger.Mary.Value (Value (..), policies, policyID)
+import qualified Cardano.Ledger.Mary.Value as Mary
+import Cardano.Ledger.SafeHash
+  ( HashAnnotated,
+    SafeHash,
+    SafeToHash,
+    extractHash,
+    unsafeMakeSafeHash,
+  )
+import Cardano.Ledger.Shelley.CompactAddr (CompactAddr, compactAddr, decompactAddr)
+import Cardano.Ledger.Shelley.Delegation.Certificates (DCert)
+import Cardano.Ledger.Shelley.PParams (Update)
+import Cardano.Ledger.Shelley.Scripts (ScriptHash (..))
+import Cardano.Ledger.Shelley.TxBody (Wdrl (Wdrl), unWdrl)
+import Cardano.Ledger.ShelleyMA.Timelocks (ValidityInterval (..))
+import Cardano.Ledger.TxIn (TxIn (..))
+import Cardano.Ledger.Val
+  ( DecodeNonNegative,
+    Val (..),
+    adaOnly,
+    decodeMint,
+    decodeNonNegative,
+    encodeMint,
+    isZero,
+  )
+import Control.Monad (guard)
+import Data.Bits
+import Data.Coders
+import Data.Maybe (fromMaybe)
+import Data.MemoBytes (Mem, MemoBytes (..), memoBytes)
+import Data.Sequence.Strict (StrictSeq)
+import qualified Data.Sequence.Strict as StrictSeq
+import Data.Set (Set)
+import qualified Data.Set as Set
+import Data.Typeable (Proxy (..), Typeable, (:~:) (Refl))
+import Data.Word
+import GHC.Generics (Generic)
+import GHC.Records (HasField (..))
+import GHC.Stack (HasCallStack)
+import GHC.TypeLits
+import NoThunks.Class (InspectHeapNamed (..), NoThunks)
+import Prelude hiding (lookup)
+
+data TxOut era
```

Comment:
Where is the burn address here?

Reply:
oh, i spoke with jared and he said that we aren't adding the burn address anymore

Reply:
that correct, we are not implementing burn addresses in babbage